build/steps-build-ios.yml

```diff
@@ -66,7 +66,7 @@ steps:
   inputs:
     bannerVersionNameText: ${{ parameters.BannerVersionNameText }}
     bannerVersionNameColor: $(BannerVersionNameColor)
-    contents: '$(Build.SourcesDirectory)/src/app/ApplicationTemplate.Mobile/iOS/Media.xcassets/AppIcons.appiconset/Icon-*.png'
```

Comment:
well "ApplicationTemplate" is supposed to be replaced by whatever you input as the project name when using the `dotnet new` command.

Reply:
oh you're right

![image](https://github.com/nventive/UnoApplicationTemplate/assets/17461593/d2249320-4832-4bd6-adec-a0c1d35db982)
![image](https://github.com/nventive/UnoApplicationTemplate/assets/17461593/81048805-a6fe-4ab8-b6e6-a5c008303545)

Reply:
Well I think it still makes thing more dynamic and at least more consistent especially with the other task. I'm going to change the PR description

Reply:
Yeah I feel like it's better to use variable if variable value are changed base on the name.